app/src/main/java/org/simple/clinic/sync/SyncModule.kt

```diff
@@ -80,12 +81,14 @@ class SyncModule {
       prescriptionSync: PrescriptionSync,
```

Comment:
In `SyncModule`, there is another provider, `@Named("frequently_syncing_repositories")`, which needs the blood sugar repository to be added in the list of repositories.

Reply:
done